### ISSUE TITLE
Remaining examples without CodePen buttons

### ISSUE THREAD

**Issue by zcorpan** [w3c/aria-practices#1790]
I've updated this project board to move cards associated with PRs that were merged into the "Done" column:

https://github.com/w3c/aria-practices/projects/31

I can see 2 open PRs that add CodePen buttons to more examples: #1651, #1685.

The "Feed" pattern is in the "Pages that are blocked" column, but not comment for why. Reviewing the example, the feed pattern is on a separate page instead of being included in the example page itself.

When doing a static analysis of which example pages *don't* contain a CodePen button (or rather, don't contain the substring `example-heading`), I found several that are not covered by the above-mentioned open PRs:

```
$ find . -name "*.html" | xargs grep -L 'example-header'
```

- ./disclosure/disclosure-navigation.html (PR #1651 -- has unresolved issues)
- ./index.html _(not an example)_
- ./menubar/menubar-editor.html
- ./treeview/treeview-1/treeview-1b.html (PR #1685)
- ./treeview/treeview-1/treeview-1a.html (PR #1685)
- ./js/notice.html _(not an example)_
- ./toolbar/help.html
- ./toolbar/toolbar.html (PR #1685)
- ./dialog-modal/datepicker-dialog.html (PR #1685)
- ./landmarks/complementary.html
- ./landmarks/HTML5.html
- ./landmarks/form.html
- ./landmarks/index.html
- ./landmarks/region.html
- ./landmarks/contentinfo.html
- ./landmarks/at.html
- ./landmarks/resources.html
- ./landmarks/banner.html
- ./landmarks/main.html
- ./landmarks/search.html
- ./landmarks/navigation.html
- ./feed/feed.html
- ./feed/feedDisplay.html
- ./coding-template/Depricated-MultipleImplementationExample-Template.html _(remove this file?)_
- ./grid/advancedDataGrid.html

Are these intentionally omitting the CodePen buttons, or are there cards in the [project board](https://github.com/w3c/aria-practices/projects/31) that should be moved back from "Done" to "Backlog"?

cc @jesdaigle

**Comment by mcking65** [comment]
@zcorpan thank you for the validation of progress and update of the project board.

We are not going to do any of the /landmark example pages.

Advanced data grid is just a placeholder page; still don't have an example there.

Feed is blocked for now.

I think that leaves only editor menubar not covered by an open PR. I moved editor menubar back to the backlog column. We mistakenly had it covered by pr #1741, which was for re-write of navigation menubar.

**Comment by zcorpan** [comment]
Current status: the backlog and in progress columns are now empty. https://github.com/w3c/aria-practices/pull/1876 and https://github.com/w3c/aria-practices/pull/1651 are in review. The feed example is blocked.

What is blocking the feed example?

**Comment by mcking65** [comment]
Since we are not implementing codepen for landmark examples and since feed will be unblocked and resolved by #2775, closing this issue.
